src/tools/compiletest/src/runtest.rs

```diff
@@ -1942,6 +1942,7 @@ impl<'test> TestCx<'test> {
                 rustc.args(&[
                     "-Zdump-mir=all",
                     "-Zmir-opt-level=3",
+                    "-Zvalidate-mir",
```

Comment:
Good idea 👍

Reply:
I didn't see this one, isn't that too expensive?

Reply:
Oh, it's only for mir-opt tests. Yes that makes sense. :)